packages/core/src/testWeb/shared/utilities/timeoutUtils.test.ts

```diff
@@ -0,0 +1,394 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * IMPORTANT: This test is a copy of of the Node timeoutUtils.test.ts file, with
+ * all tests in the `describe('timeoutUtils')` block.
+ * We should look to keep the tests 1:1 so that updates to the Node test file
+ * can be easily copied here.
```

Comment:
can this file import the non-web test instead?

Reply:
Good point, was able to export the `describe` and it all works